mods/gen4/abilities.js

```diff
@@ -59,6 +59,18 @@ exports.BattleAbilities = {
 	},
 	"flashfire": {
 		inherit: true,
+		onTryHit: function (target, source, move) {
+			if (target !== source && move.type === 'Fire') {
+				move.accuracy = true;
+				if (target.status === 'frz') {
+					return;
+				}
```

Comment:
Out of interest, is that the right place for the accuracy bypass?

Reply:
didnt see this until now, whoops